app/utils.py

```diff
@@ -58,6 +58,40 @@ def pagination_links(pagination, endpoint, args):
     return links
 
 
+def result_meta(total_count):
+    return {"total": total_count}
+
+
+def single_result_response(result_name, result, serialize_kwargs={}):
+    """Return a standardised JSON response for a single serialized SQLAlchemy result e.g. a single brief"""
+    return jsonify(**{result_name: result.serialize(**serialize_kwargs)})
+
+
+def list_result_response(result_name, results_query, serialize_kwargs={}):
+    """
+    Return a standardised JSON response for a SQLAlchemy result query e.g. a query that will retrieve closed briefs.
+    The query should not be executed before being passed in as a argument so we can manipulate the query object (i.e. to
+    get the count). Results will be returned in a list and use the results `serialize` method for presentation.
+    """
+    meta = result_meta(results_query.count())
```

Comment:
@risicle Was just checking if you had any thoughts about my use of `count` here? It will trigger another call to the DB. 

Two alternatives:
1. Pass in a result query as we are currently doing but use use python to count the results after they have been retrieved.
2. Pass in a list of results instead of a result query (similar to how we do for a single result rather than how we do for pagination) and do the counting in python again

I think I prefer the current solution but that opinion is lightly held.

Reply:
Why do we need to give a count in this case at all? I thought the purpose of a count was to allow you to see the number of items when most of them are paged away. If we're getting a list of all of them, well, the consumer can just count them themselves, surely? If it's just for completeness/api symmetry then yeah, count them in python.

Reply:
We are generally doing it on this type of response already (it also feels slightly helpful when doing dev work) so I'd be keen to keep it.

I'll swap and use python to count.

Reply:
If we're doing this already and incurring an unnecessary db hit just to get the count then my mouth is agape.